### DISCUSSION TITLE
Whenever I run the command dart run sqflite_common_ffi_web:setup to generate the files, it generates the error below.

### DISCUSSION THREAD

**Discussion by mgdonato23** [tekartik/sqflite#1121] [Show and tell]
[SEVERE] build_web_compilers:entrypoint on web/sqflite_sw.dart: ExitCode:1
StdOut:
org-dartlang-app:///packages/sqflite_common_ffi_web/src/web/load_sqlite_web.dart@2157+1:
Error: The argument type 'String' can't be assigned to the parameter type 'JSAny'.
org-dartlang-app:///packages/sqflite_common_ffi_web/src/web/load_sqlite_web.dart@2451+1:
Error: The argument type 'String' can't be assigned to the parameter type 'JSAny'.
Error: Compilation failed.

StdErr:

[INFO] Running build completed, took 35.7s
[INFO] Caching finalized dependency graph completed, took 448ms
[SEVERE] Failed after 36.2s
[SEVERE] FailureType: 1
Unhandled exception:

**Comment by alextekartik** [comment]
There is indeed an unfortunate breaking change in web 1.0.0 that prevents compatibility for the setup tool.

I can see 2 workarounds until your project supports web 1.0.0

### Solution 1: Temp overrides

Temporarily overrides the dependency - just to setup the workers.

```
# Temporarily add the overrides
flutter pub add 'override:sqflite_common_ffi_web:>=0.4.4' override:web:^1.0.0
# Setup the worker
flutter pub run sqflite_common_ffi_web:setup --force
# Remove the overrides
flutter pub remove override:sqflite_common_ffi_web override:web
```

### Solution 2: Create the worker in another project

The workaround here is to run the setup in another project and copy the generated `web/sqlite3.wasm`  and `web/sqflite_sw.js`

```
dart create -t web tmpsqfliteweb --force
cd tmpsqfliteweb
dart pub add sqflite_common_ffi_web
dart pub run sqflite_common_ffi_web:setup
```
